bundles/remote_services/civetweb/src/civetweb.c

```diff
@@ -1,4 +1,4 @@
-	/* Copyright (c) 2013-2015 the Civetweb developers
+/* Copyright (c) 2013-2015 the Civetweb developers
```

Comment:
While I agree with the changes, I think this should not be done here. Either upstream, or a fork of civetweb (IMO).

@pnoltes wdyt?

Reply:
I prefer civetweb accept the changes, but I am not sure if that going to happen. 

Seeing that we have already have a copy of the civetweb sources in the celix source tree, I think it is ok for now. 

If civetweb does not update their sources, it is maybe time to look for an alternative...

Reply:
Some changes have been made according to the [conversation](https://github.com/civetweb/civetweb/issues/861#issuecomment-639315880), but some remain. Though I'd say that we should strive for no threadsanitizer warnings, the one they leave in seems to be reasonably benign.

Reply:
Can you update with upstream so that we are as close as possible to upstream if we add our own changes?

Reply:
I'd like to update civetweb in a different pull request.